### DISCUSSION TITLE
Debug on M1 Apple silicon Someday?

### DISCUSSION THREAD

**Discussion by jesusdeveloper** [microsoft/vscode-cpptools#8073] [Q&A]
Hello!

I have tried trying to debug a cpp with Visual Studio Code and Microsoft's C / C ++ extension but it is not supported.
You have to install lldb (although it doesn't debug me either).
I wonder if the next updates plan to incorporate compatibility to be able to debug cpp with computers with m1 apple silicon processor.

Thank you very much,  a greetings!

**Comment by bobbrow** [comment]
We are patiently waiting for .NET 6 to be released so we can support this. Feel free to add a 👍 to this issue: https://github.com/microsoft/vscode-cpptools/issues/7035
